src/main/java/com/acrolinx/sidebar/lookup/LookupForResolvedEditorViews.java

```diff
@@ -181,6 +191,28 @@ private void lookupMatchInContentNode(LookupForResolvedViewsHelper utils, List<?
         });
     }
 
+    private int findFragmentStartOffsetInCurrentDocument(ContentNode contentNode, AbstractMatch match)
+    {
+        final int contentFragmentLength = contentNode.getAsXMLFragment().length();
+        final int matchStartOffset = match.getRange().getMinimumInteger();
+
+        if (contentFragmentLength < matchStartOffset) {
+            return matchStartOffset - contentFragmentLength;
+        }
+        return matchStartOffset;
```

Comment:
After looking a bit closely I think there is an issue with the default return.
We should return `0` and not `matchStartOffset`

Reason: Returning `matchStartOffset` will make diff algorithm to run just on exact matched word

Reply:
made the change 👍🏻